save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/entities/benchmarks/TabMenuBar.kt

```diff
@@ -0,0 +1,58 @@
+package com.saveourtool.save.entities.benchmarks
```

Comment:
`entities` is used for DTOs mostly that have corresponding `@Entity` class is `jvmMain`. Please move this file to something like `com.saveourtool.save.frontend`